website/client/src/pages/private-messages.vue

```diff
@@ -691,6 +737,31 @@ export default {
         description: this.$t('PMPlaceholderDescription'),
       };
     },
+    disabledTexts () {
+      if (this.user.flags.chatRevoked) {
+        return {
+          title: this.$t('PMPlaceholderTitleRevoked'),
+          description: this.$t('chatPrivilegesRevoked'),
+        };
+      }
+
+      if (this.user.inbox.optOut) {
+        return {
+          title: this.$t('PMDisabledCaptionTitle'),
+          description: this.$t('PMDisabledCaptionText'),
+        };
+      }
+
+      console.info(this.selectedConversation);
```

Comment:
Looks like this is breaking the build on delta, can you remove it?

Reply:
oof sorry, removed! I wonder why this check isn't added on the current CI